sql database/scripts/createTrain.py

```diff
@@ -0,0 +1,76 @@
+# create a training set from annotated images
+USAGE = "python export db_path dest_path [query]"
+
+import shutil, os
+import sys
+import sqlite3
+from sqlite3 import Error
+import pandas as pd
+import numpy as np
+import json
+
+# create a connection
+def create_connection(db_file):
+	""" create a db connection to database specified
+		by db_file
+	:param db_file: database file path
+	:return: Connection object or None
+	"""
+	try:
+		conn = sqlite3.connect(db_file);
+		return conn
+	except Error as e:
+		print(e)
+
+	return None
+
+# select all images in the database
+def select_all_images(conn):
+	"""	Query all rows in the images tables
+	:param conn: the Connection object
+	:return: list of rows
+	"""
+	cur = conn.cursor()
+	cur.execute("SELECT Filename FROM images")
+
+	return cur.fetchall()
+
+# select all images with a certain species
+def select_image_by_query(conn, query):
+	"""	Query image by species
+	:param conn: the Connection Object
+	:param query: manual query
+	:return: all rows of the species
+	"""
+	cur = conn.cursor()
+	cur.execute(query)
+
+	return cur.fetchall()
+
+
+# Execution entrypoint
+if __name__ = '__main__':
+	
+	# check for args
+	if len(sys.argv) <= 2:
+		print(USAGE)
+		exit()
+	# declare cl arguments
+	db = sys.argv[1]
```

Comment:
A better way to declare CL arguments is probably argparse or cli since it provides documentation features than sys.argv. It's pretty easy to use. 
#### sys-argv vs argparse
https://stackoverflow.com/questions/35365344/python-sys-argv-and-argparse

#### comparison of different parsing libraries
just additional resource if you want to look at others
https://realpython.com/comparing-python-command-line-parsing-libraries-argparse-docopt-click/